config/projects.yml

```diff
@@ -80,6 +80,13 @@ classification-bundle:
         symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
         sonata_admin: ['3']
 
+classification-media-bundle:
+  branches:
+    master:
+      php: ['5.6', '7.0']
```

Comment:
> No, because the new bundle would only support the latest master/4.0 version of the dependencies.

This is especially what I'm talking about. :-)

Moving code to a separated bundle is BC break, right? I think this can be BC with ease:

* Move the code (already done)
* Deprecate the copied classes on 3.x and require the new bundle (BC part)
* Then on 4.x, remove the classes